src/domain/help/pages/termsOfUsePage/TermsOfUsePage.tsx

```diff
@@ -64,27 +141,106 @@ const TermsOfUsePage: React.FC = () => {
+              Linked Events -palveluun kirjaudutaan Helsingin kaupungin
+              Tunnistamo-sovelluksella, jossa kirjautumisvaihtoehtoina ovat:
+            </p>
+            <ul className={styles.termOfUse}>
+              <li>Helsingin kaupungin AD</li>
+              <li>Facebook</li>
+              <li>Google</li>
+              <li>Yle</li>
+            </ul>
+            <p>
+              Tieto Linked Eventsiin kirjautumisesta tallentuu näiden
+              palvelutarjoajien tietoihin.
+            </p>
+            <p>Linked Events kerää tapahtumailmoittajista seuraavat tiedot:</p>
+            <ul className={styles.termOfUse}>
+              <li>Etu- ja sukunimi</li>
+              <li>Sähköpostiosoite</li>
+              <li>
+                Mitä organisaatiota/yrityksiä/yhteisöjä/järjestöjä käyttäjä
+                edustaa
+              </li>
+              <li>Milloin käyttäjä kirjautuu palveluun ensimmäistä kertaa</li>
+            </ul>
+            <p>
+              Tiedot tallennetaan Linked Events -tietokantaan, eikä niitä
+              välitetä rajapinnan kautta eteenpäin. Itse palvelussa saman
+              organisaation admin-käyttäjät näkevät tapahtuman ilmoittaneen
+              sähköpostiosoitteen. Kaupunkiorganisaation ulkopuolisten osalta
+              ilmoittajan sähköpostiosoitteen näkee moderaattori, joita on 2-5
+              riippuen moderoitavasta kokonaisuudesta.
+            </p>
+            <p>
+              Käyttöoikeuksien hallinnan osalta tapahtumailmoittajien tiedot
+              näkevät kaupungin Linked Events -pääkäyttäjät (2-5 henkilöä) sekä
+              kehitystiimit.
+            </p>
+            <p>
+              Linked Events -palvelu sijaitsee pilvipalvelussa Helsingin
+              kaupungin Azure Openshift -alustalla, ja käyttäjätietoja sisältävä
+              tietokanta on suojattu salausavaimin.
+            </p>
+            <p>
+              Tapahtumatietojen syöttämiseen tarvittavat tiedot kerätään
+              käyttäjän itsensä antamana palveluun kirjauduttaessa.
+              Sisällöntuotanto-oikeuksien osalta käyttäjä itse pyytää oikeuksia
+              joko palvelusta löytyvällä lomakkeella tai sähköpostitse
+              noudattaen EU:n yleisen tietosuoja-asetuksen 5 artiklan 1 a- ja
+              c-kohtaa. Käyttäjä on antanut luvan henkilötietojensa käsittelyyn
+              tai käsittely on tarpeen rekisterinpitäjän eli Helsingin kaupungin
+              lakisääteisen velvoitteen noudattamiseksi.
+            </p>
+            <p>
+              Tietoja säilytetään pääsääntöisesti 5 vuotta, mutta pyynnöstä ne
+              poistetaan nopeammin.
+            </p>
+            <p>
+              Helsingin kaupungin käyttöoikeuksien tietosuojaseloste
+              {/*eslint-disable*/}
+              <ExternalLink href="https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/Sahkoisten%20asiointipalveluiden%20rekisteri.pdf">
```

Comment:
<img width="566" alt="Screenshot 2023-08-25 at 14 31 01" src="https://github.com/City-of-Helsinki/linkedcomponents-ui/assets/66477579/2ed60ac6-5378-4ec2-8d63-83168f7c44a7">

Looks like links are missing space between the word before. Also I think the second "Tietosuojaseloste" can be removed here 🤔

Reply:
Added space before links